DNN Platform/Tests/DotNetNuke.Tests.Web/Api/Internals/WebFormsServiceProviderTests.cs

```diff
@@ -0,0 +1,145 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+namespace DotNetNuke.Tests.Web.Api.Internals
+{
+    using System.Web.UI;
+
+    using DotNetNuke.Abstractions;
+    using DotNetNuke.Abstractions.Application;
+    using DotNetNuke.Common;
+    using DotNetNuke.Common.Extensions;
+    using DotNetNuke.Entities.Modules;
+    using DotNetNuke.Tests.Utilities;
+    using DotNetNuke.Web.Common;
+    using Microsoft.Extensions.DependencyInjection;
+    using Moq;
+    using NUnit.Framework;
+
+    public class WebFormsServiceProviderTests
```

Comment:
Could we add some performance tests in this `TestFixture` or create a new one. Maybe hook into the `DotNetBenchmarking` tool. I am very concerned about how this change affects the overall performance of DNN.